operations/mimir-mixin/dashboards/ruler-reads.libsonnet

```diff
@@ -0,0 +1,159 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+local filename = 'mimir-reads.json';
+
+(import 'dashboard-utils.libsonnet') {
+  [filename]:
+    ($.dashboard('Ruler reads') + { uid: std.md5(filename) })
+    .addClusterSelectorTemplates()
+    .addRowIf(
+      $._config.show_dashboard_descriptions.reads,
+      ($.row('Ruler reads dashboard description') { height: '175px', showTitle: false })
+      .addPanel(
+        $.textPanel('', |||
+          <p>
+            This dashboard shows health metrics for the ruler read path.
+            It is broken into sections for each service on the ruler read path, and organized by the order in which the read request flows.
+            <br/>
+            For each service, there are 3 panels showing (1) requests per second to that service, (2) average, median, and p99 latency of requests to that service, and (3) p99 latency of requests to each instance of that service.
+          </p>
+        |||),
+      )
+    )
+    .addRow(
+      ($.row('Headlines') +
+       {
+         height: '100px',
+         showTitle: false,
+       })
+      .addPanel(
+        $.panel('Instant queries / sec') +
```

Comment:
I'm not sure we should have it. It looks a bit deceiving to me. What we're really execution are rule evaluations.

Reply:
Agree. I think the line `Evaluations / sec` would be more meaningful here.